wagtail/embeds/blocks.py

```diff
@@ -57,5 +61,10 @@ def value_from_form(self, value):
         else:
             return EmbedValue(value)
 
+    def clean(self, value):
+        if isinstance(value, EmbedValue) and not value.html:
+            raise ValidationError(_('Please enter a valid URL'))
```

Comment:
Could we reword this message? It says that the URL is invalid, but actually, it is valid but doesn't point to anything.

Reply:
I just reused the message from the rich text embed feature.

Reply:
The only place `"Please enter a valid URL"` is used is https://github.com/wagtail/wagtail/blob/1247565d3da6b542f97b59d7fd62c8c7c624f6b8/wagtail/embeds/forms.py#L7-L12 , which _is_ checking URL validity. For unrecognised embed providers, the message is `"Cannot find an embed for this URL."` - I'll update this when I merge.